static/app/components/profiling/profilingTransactionHovercard.tsx

```diff
@@ -0,0 +1,301 @@
+            functions.data.functions.map(f => {
+              const [exampleProfileIdRaw] = f.examples;
+              const exampleProfileId = exampleProfileIdRaw.replaceAll('-', '');
+              return (
+                <Fragment key={f.name}>
+                  <FunctionsMiniGridCell title={f.name}>
+                    <TextTruncate>
+                      <Link
+                        to={linkToFlamechartRoute(exampleProfileId, {
+                          frameName: f.name,
+                          framePackage: f.package,
+                        })}
+                      >
+                        {f.name}
+                      </Link>
+                    </TextTruncate>
+                  </FunctionsMiniGridCell>
+                  <FunctionsMiniGridCell align="right">
+                    <PerformanceDuration nanoseconds={f.p99} abbreviation />
+                  </FunctionsMiniGridCell>
+                  <FunctionsMiniGridCell align="right">
+                    <NumberContainer>{f.count}</NumberContainer>
+                  </FunctionsMiniGridCell>
+                </Fragment>
+              );
+            })}
+        </FunctionsMiniGrid>
+        {functions.type === 'loading' && (
+          <Flex align="stretch" justify="center" column h="100%">
+            <Flex align="center" justify="center">
+              <LoadingIndicator mini />
+            </Flex>
+          </Flex>
+        )}
+
+        {functions.type === 'resolved' && functions.data.functions.length === 0 && (
+          <Flex align="stretch" justify="center" column h="100%">
+            <Flex align="center" justify="center">
+              {t('No functions data')}
+            </Flex>
+          </Flex>
+        )}
+      </Flex>
+    </Flex>
+  );
+}
+
+interface ContextDetailProps {
+  children: React.ReactNode;
+  isLoading: boolean;
+  title?: React.ReactNode;
+}
+function ContextDetail(props: ContextDetailProps) {
+  const {title, children, isLoading} = props;
+
+  return (
+    <Flex column gap={space(1)}>
+      {title && <ContextTitle>{title}</ContextTitle>}
+      <Fragment>
+        {isLoading ? (
+          <Flex align="center" justify="center" h="1em">
+            <LoadingIndicator mini />
+          </Flex>
+        ) : (
+          children
+        )}
+      </Fragment>
+    </Flex>
+  );
+}
+
+const px = (val: string | number | undefined) =>
+  typeof val === 'string' ? val : typeof val === 'number' ? val + 'px' : undefined;
+
+// TODO(@eliashussary): move to common folder / bring up in fe-tsc
+const Flex = styled('div')<{
+  align?: CSSProperties['alignItems'];
+  column?: boolean;
+  gap?: number | string;
+  h?: number | string;
+  justify?: CSSProperties['justifyContent'];
+  maxH?: number | string;
+  minH?: number | string;
+  w?: number | string;
+}>`
+  display: flex;
+  flex-direction: ${p => (p.column ? 'column' : 'row')};
+  justify-content: ${p => p.justify};
+  align-items: ${p => p.align};
+  gap: ${p => px(p.gap)};
```

Comment:
I think the preference is to use space() helper so preserve the scales

Reply:
agreed; i left it up the caller in this case, if you have a look where i'm using `gap` i'm using `space()` 

i can make it a first class part of this component ... was hoping to make those decisions at a later time when/if we make this a more permanent fixture of our component lib

ideally it becomes a hand off to `core` lol